Makefile

```diff
@@ -159,6 +159,9 @@ deploy:
         done
 	@echo
 	@echo
+	while ! kubectl get services | grep kubedirector-validator; do \
+	    sleep 3; \
+	done
```

Comment:
Yeah this is a really tight window you're hitting, if the webhook configuration exists but the service is not created yet. However there is in fact a window there and I guess you're hitting it!

The only tweak I would recommend is just to use the same form of "does this thing exist" check as the other checks in the Makefile. E.g.:
```
	while ! kubectl get service kubedirector-validator &> /dev/null; do \
	    sleep 3; \
	done
```

I guess we could get fancier with a bailout after some time, but really if the webhook configuration exists then the service SHOULD be along really soon.